app/src/ui/add-repository/write-default-readme.ts

```diff
@@ -3,18 +3,21 @@ import * as Path from 'path'
 
 const DefaultReadmeName = 'README.md'
 
-function defaultReadmeContents(name: string): string {
-  return `# ${name}\n`
+function defaultReadmeContents(name: string, description?: string): string {
+  return description !== undefined
+    ? `# ${name}\n ${description}`
```

Comment:
Forgot to mention that we should put a newline after the description:

`# ${name}\n ${description}\n`